bottom-navigator/src/test/java/androidx/fragment/app/FakeFragmentManager.kt

```diff
@@ -19,14 +19,16 @@ import android.os.Bundle
 import android.view.View
 import java.util.ArrayList
 
-class FragmentState(var fragment: Fragment, var attached: Boolean, var shown: Boolean)
+class FragmentState(var fragment: Fragment, var attached: Boolean, var shown: Boolean,
```

Comment:
Since FragmentState holds the current state of the fragment, it's a logical place to record the transitions that executed when that fragment was added or shown. In FakeFragmentTransaction, I removed setCustomAnimations(...) so we call the super's method and set the transaction's enterAnim and exitAnim. Then, when a fragment is added or shown, and we update FragmentState, we also save the transaction's animations.